app/Export/ImageConversion.hs

```diff
@@ -18,7 +18,7 @@ createImageFile inName outName =  do
     _ <- waitForProcess pid
     putStrLn "Process Complete"
 
--- | Converts an SVG file to a PNG file. Note that image magik's 'convert' command
+-- | Converts an SVG file to a PNG file. Note that image magik's 'magick' command
```

Comment:
This is good, but since you're already changing this can you also please change "image magik" to "ImageMagick" in the comment? I didn't spot that in the initial commit, sadly.